Delete_User.css

```diff
@@ -92,7 +100,7 @@ ul li:hover ul{
 
 
 .img_2{
-	background-color: green;
+	background-color: white;
```

Comment:
I change the background img in the right side of the add user to be white because I think if for some reason the img is take more time to load or can't be display then that area will be better with the color white then the color green.